pkg/cloudprovider/aws/aws.go

```diff
@@ -219,6 +224,27 @@ func (n *NodeGroup) Belongs(node *v1.Node) bool {
 	return false
 }
 
+// NodeInstantiationTime retrieves the instantiation time of the backing instance for the node
+func (c *CloudProvider) NodeInstantiationTime(node *v1.Node) time.Time {
+	var launchTime time.Time
+	instanceIds := make([]*string, 0, 1)
+	// TODO PdZ - Do we need to wrap the following split to avoid crashes from bad data?
```

Comment:
We already have some logic to determine the EC2 ID from the Kubernetes node object here - https://github.com/atlassian/escalator/blob/master/pkg/cloudprovider/aws/aws.go#L185-L192

This should pulled out and turned into a reusable internal function inside the AWS cloud provider struct.